libkbfs/folder_branch_ops.go

```diff
@@ -4160,13 +4197,22 @@ const (
 const CtxFBOOpID = "FBOID"
 
 func (fbo *folderBranchOps) ctxWithFBOID(ctx context.Context) context.Context {
-	return ctxWithRandomID(ctx, CtxFBOIDKey, CtxFBOOpID, fbo.log)
+	return ctxWithRandomIDReplayable(ctx, CtxFBOIDKey, CtxFBOOpID, fbo.log)
+}
+
+func (fbo *folderBranchOps) newCtxWithFBID() (context.Context, context.CancelFunc) {
+	ctx := NewContextReplayable(context.Background(), fbo.ctxWithFBOID)
```

Comment:
Actually I think you were right with your first (deleted) comment! This attaches as a replayable function, which means it gets executed every times it replays. Although in `ctxWithRandomIDReplayable` it attaches a reaplayable using "static" value, that generation of "static" value is itself in replayable as well, causing the FBOID set for multiple times.

So this should instead be just `ctx := fbo.ctxWithFBOID()`. Does it make sense?